File: nelio-alves-algoritmo-e-logica-de-programacao/java/src/exercicios/DentroFora.java
package exercicios;

import java.util.Locale;
import java.util.Scanner;

public class DentroFora {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int N, X, fora, dentro;
		fora = 0;
		dentro = 0;

	    System.out.print("Quantos numeros voce vai digitar? ");
	    N = sc.nextInt();

		for (int i = 0; i < N; i++) {
	    	System.out.print("Digite um numero: ");
	        X = sc.nextInt();

			if ((X >= 10) && (X <= 20)) {
				dentro++;
			} else {
				fora++;
			}
	    }
		
	    System.out.printf("%d DENTRO\n", dentro);
	    System.out.printf("%d FORA\n", fora);

		sc.close();
	}
}

File: nelio-alves-algoritmo-e-logica-de-programacao/c/exercicios/dentro_fora.c
#include <stdio.h>

int main() {
    int N, X, fora, dentro;
    fora = 0;
    dentro = 0;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &N);

    for (int i = 0; i < N; i++) {
        printf("Digite um numero: ");
        scanf("%d", &X);

        if ((X >= 10) && (X <= 20)) {
            dentro++;
        } else {
            fora++;
        }
    }

    printf("%d DENTRO\n", dentro);
    printf("%d FORA\n", fora);

    return 0;
}
